doc/api/util.md

```diff
@@ -70,6 +70,28 @@ callbackFunction((err, ret) => {
 });
 ```
 
+## `util.callWithTimeout(timeout, fn)`
+<!-- YAML
+added: REPLACEME
+-->
+
+* `timeout` {integer} Specifies the number of milliseconds to execute `fn`
+  before terminating execution. If execution is terminated, an Error will be
+  thrown. This value must be a strictly positive integer.
+* `fn` {Function} The function to invoke with a timeout.
+
+The `util.callWithTimeout` method allows interrupting synchronous execution
+after a set amount of time. This method should not be used to untrusted code.
+Calls into native addons may, in some cases, exceed the timeout.
+
+```js
+const { callWithTimeout } = require('util');
+
+callWithTimeout(10, () => {
+  while (true) {}
+});
```

Comment:
It's a bit bike-sheddy but looking at this my initial reaction is to prefer swapping the order of the arguments so that it's more like `setTimeout()` or `setInterval()`...

```js
callWithTimeout(() => { while (true) {} }, 10)
```

Reply:
This should clarify that the timeout only impacts the synchronous execution of the code. The timeout does not cover, for instance, the complete execution a resolution of an async function that may be passed in.